src/rgw/rgw_cr_rest.h

```diff
@@ -35,6 +39,7 @@ class RGWReadRESTResourceCR : public RGWSimpleCoroutine {
     if (ret < 0) {
       log_error() << "failed to send http operation: " << op->to_str()
           << " ret=" << ret << std::endl;
+      op->put();
```

Comment:
oh, gross! `intrusive_ptr()`'s ctor takes a second `bool add_ref = true` argument, let's pass false to keep it at 1 then